### PR TITLE
Arrows leading to ContactPostStaff and ContactOtherStaff activities as shown in mockup

### PR DESCRIPTION
**Pull request by rohands**
screenshot_2016-03-02-03-07-49
screenshot_2016-03-02-03-08-38

### CODE REVIEW

**Comment by rohands** [comment]
Please ignore the comment message along with the commit

**Comment by sandarumk** [comment]
Please add meaningful commit messages when committing.
http://stackoverflow.com/questions/15324900/standard-to-follow-when-writing-git-commit-messages

**Comment by rohands** [comment]
@sandarumk Should I PR this again with the right message?

**Comment by sandarumk** [comment]
Yeah sure.

**Comment by sandarumk** [comment]
Apart from the commit message the rest looks fine.

File: app/src/main/res/layout/activity_reporting_contact_other_staff.xml [hunk 1]
```diff
@@ -47,29 +47,65 @@
...
-            android:id="@+id/oig"
-            layout="@layout/button"/>
-        <include
-            android:id="@+id/ocrd"
-            layout="@layout/button"/>
+       <LinearLayout
+           android:layout_width="match_parent"
+           android:layout_height="wrap_content"
+           android:orientation="horizontal"
+           android:layout_marginTop="30dp">
```
Comment (Buddhiprabha) [inline app/src/main/res/layout/activity_reporting_contact_other_staff.xml]: Can you add this and other dimensions to dimens.xml?

Comment (sandarumk) [inline app/src/main/res/layout/activity_reporting_contact_other_staff.xml]: And rest of the dimensions which you have use the numbers too.

**Comment by rohands** [comment]
#87 solves the problem
Sorry for creating another PR altogether, had some issues with the fork.

**Pull request closed without merging**